providers/capv/standard/capv_test.go

```diff
@@ -8,6 +8,7 @@ import (
 
 var _ = Describe("Common tests", func() {
 	common.Run(&common.TestConfig{
-		BastionSupported: false,
+		BastionSupported:     false,
+		ExternalDnsSupported: false,
```

Comment:
Maybe something like this so we won't forget

```
// TODO <Link to issue> We want to deploy external-dns as
// default app to CAPV at some point. Until then, related tests
// are disabled. There's an expiry date for this flag so we notice that we'll have to fix this.
ExternalDnsSupported: time.Now().After(time.Date(2024, 3, 1, 0, 0, 0, 0, time.UTC))
```

Reply:
After asking around, support for `external-dns` is part of a big story in provider alignments around DNS. I don't think we have a date for that, and I don't think we should have a deadline for it. I think linking to an issue definitely makes sense.